code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -181,7 +181,7 @@
 
 
 /obj/item/weapon/reagent_containers/syringe/update_icon()
-	var/rounded_vol = min(max(round(reagents.total_volume,5),5),15)
+	var/rounded_vol = round(reagents.total_volume,5)

```

Comment:
Shouldn't this be `Clamp(round(reagents.total_volume,5), 0, 15)` instead of just removing the sanity entirely? Though I guess this would only give a broken icon_state if we had a bad amount of reagents in a syringe, but either way.